tests/Model/Command/ItemPropertyTest.php

```diff
@@ -2,10 +2,20 @@
 
 namespace Lmc\Matej\Model\Command;
 
+use Lmc\Matej\Exception\DomainException;
 use PHPUnit\Framework\TestCase;
 
 class ItemPropertyTest extends TestCase
 {
+    /**
+     * @test
+     */
+    public function shouldNotAllowItemIdInProperties(): void
+    {
+        $this->expectException(DomainException::class);
```

Comment:
I'd add expectExceptionMessage - to make sure the exception occurred in the expected case.